runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewDelegate.java

```diff
@@ -161,8 +161,13 @@ public static void init(XWalkViewInternal xwalkView) throws UnsatisfiedLinkError
         }
 
         ResourceExtractor.setMandatoryPaksToExtract(MANDATORY_PAKS);
-        final int resourcesListResId = context.getResources().getIdentifier(
-                XWALK_RESOURCES_LIST_RES_NAME, "array", context.getPackageName());
+        int resListResId = context.getResources().getIdentifier(
+                XWALK_RESOURCES_LIST_RES_NAME, "array", context.getClass().getPackage().getName());
+        if (resListResId == 0) {
+            resListResId = context.getResources().getIdentifier(
```

Comment:
I cannot tell you the exact situation.  It seems that different tools do the renaming differently.  See this change from Cordova: https://issues.apache.org/jira/browse/CB-5481
They found that they had to do it both ways to have it work correctly with aapt and gradle builds.  We also have to have it work correctly with Mocana ( https://www.mocana.com/ ).  After seeing what worked for Cordova I took their approach.  I know that it works correctly with the tools just mentioned.